src/blockchain/projects.js

```diff
@@ -164,7 +164,7 @@ const projects = (app, liquidPledging) => {
     }
 
     if (data.length > 1) {
-      logger.error('more then 1 milestone with the same txHash found: ', data);
+      logger.error('more then 1 milestone with the same txHash found: ', data.length);
```

Comment:
The same as above

Reply:
@aminlatifi  done.
but  for shadowing variable names(eslint error) in two cases I used the projectId declared previously, that should not affect the code.